src/webui/webui.cpp

```diff
@@ -93,7 +94,7 @@ void WebUI::init()
 #endif
 
         if (!m_httpServer->isListening()) {
-            bool success = m_httpServer->listen(QHostAddress::Any, m_port);
+            bool success = m_httpServer->listen(address, m_port);
```

Comment:
just inline the call

Reply:
I put it early so that the exception would be thrown as early as possible to report the error. But now if we just returning an empty QHostAddress on error this makes sense.